squirrels/_package_data/base_project/pyconfigs/user.py

```
@@ -15,25 +16,36 @@ class CustomUserFields(auth.CustomUserFields):
     role: Literal["manager", "staff", "customer"] = "staff"
 
 
-# @auth.provider(name="google", label="Google", icon="https://www.google.com/favicon.ico")
-def google_auth_provider(sqrl: args.AuthProviderArgs) -> auth.ProviderConfigs:
+# @provider(
+#     name="google", label="Google", 
+#     icon="https://www.google.com/favicon.ico"
+#     # ^ Note: You also can save the google favicon as an image file in "resources/public/logos/google.ico" 
+#     #   and use icon="/public/logos/google.ico" instead
+# )
+def google_auth_provider(sqrl: AuthProviderArgs) -> ProviderConfigs:
```

Ruff (0.14.13)

Unused function argument: `sqrl`

(ARG001)